src/.eslintrc.yml

```diff
@@ -2,3 +2,6 @@ extends: '@travi/travi/rules/react'
 
 globals:
   graphql: true
+
+rules:
+  react/jsx-no-bind: off # render props... ?
```

Comment:
ive actually not gotten on the render-props bandwagon much yet, so i dont have a strong opinion about this. (i havent found a good way to unit test the behavior of them that i like and havent had enough need for them in my use cases yet. might be a good conversation sometime to get me up to speed on your thoughts)